examples/config-many-files/teams-config.yml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: options
+spec:
+  config:
+    team_name: default
+    options:
```

Comment:
I haven't really thought about it because I haven't had a need for it in other fleet types, but I imagine `options` will be mostly the same with minor differences across teams. With that in mind, it would be good to support YAML anchors to reduce required duplication.

I'm putting this on options, but the same thing applies for basically any piece of config here.

Good example: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

(I don't know whether or not this is currently supported in any of fleet config; if it is, can mostly ignore this comment other than maybe updating docs to reference it)

Reply:
Ah, this is the first time I've seen YAML anchors. Thank you for the examples link. I assume Fleet config support anchors but I'll try to confirm this.

>I imagine options will be mostly the same with minor differences across teams

This is very helpful information.